### PR TITLE
KSQL-758-Add an optional properties file for data-gen 

### PR DESCRIPTION
**Pull request by hjafarpour**
Added an optional properties file for data-gen that can be used to register interceptors when KSQL is used with CE.

### CODE REVIEW

File: ksql-examples/src/main/resources/datagen.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
```
Comment (big-andy-coates) [inline ksql-examples/src/main/resources/datagen.properties]: Am I right in thinking that `MonitoringProducerInterceptor` will only be available on the class path for enterprise users?  Won't this cause problems / exceptions for non-enterprise users? Or is the idea that only enterprise users would use this properties file?

Comment (hjafarpour) [reply ksql-examples/src/main/resources/datagen.properties]: Yes, this will only be used when we have CE.

Comment (dguy) [reply ksql-examples/src/main/resources/datagen.properties]: Should this be documented? Even if it is just a comment in the properties file?

Comment (hjafarpour) [reply ksql-examples/src/main/resources/datagen.properties]: Good point. I'm adding a comment in the properties file.

**Comment by hjafarpour** [comment]
@dguy @big-andy-coates @apurvam @rodesai Can I get two reviews for this so I can merge it :)

**Pull request merged**